doc/ref_impl/cntt-ri/chapters/chapter03.md

```diff
@@ -128,57 +128,57 @@ This profile is the global settings for the whole NFVI, which means there should
 | Crypto acceleration | N  | Y | Y |
 | Crypto Acceleration Interface | N  | Y | Y |--->
 
-| .conf | Basic | Network Intensive | 
+| .conf | Basic | Network Intensive |
 |-------|----------------|----------------|
-| CPU allocation ratio  | 4:1 | 1:1  | 
-| NUMA awareness | N | Y | 
-| CPU pinning capability | N | Y | 
-| Huge Pages | N | Y | 
-| Catalogue storage Types | Y  | Y  | 
+| CPU allocation ratio  | 4:1 | 1:1  |
+| NUMA awareness | N | Y |
+| CPU pinning capability | N | Y |
+| Huge Pages | N | Y |
+| Catalogue storage Types | Y  | Y  |
 | Storage Block | Y | Y |
-| Storage Object | Y | Y | 
-| Storage with replication | N | Y | 
-| Storage with encryption | Y | Y | 
-| Storage IOPS oriented | N | Y | 
+| Storage Object | Y | Y |
+| Storage with replication | N | Y |
+| Storage with encryption | Y | Y |
+| Storage IOPS oriented | N | Y |
 | Storage capacity oriented | N | N |
-| vNIC interface | virtio1.1 |  virtio1.1| 
+| vNIC interface | virtio1.1 |  virtio1.1|
 | Overlay protocol | VXLAN, MPLSoUDP, GENEVE, other |  VXLAN, MPLSoUDP, GENEVE, other |
 | NAT | Y | Y |
-| Security Group | Y | Y | 
-| SFC support | N | Y | 
-| Traffic patterns symmetry | Y | Y | 
-| vSwitch optimisation | N | Y, DPDK | 
+| Security Group | Y | Y |
+| SFC support | N | Y |
+| Traffic patterns symmetry | Y | Y |
+| vSwitch optimisation | N | Y, DPDK |
 | Support of HW offload | N | Y, support of SmartNic |
-| Crypto acceleration | N  | Y | 
-| Crypto Acceleration Interface | N  | Y | 
+| Crypto acceleration | N  | Y |
+| Crypto Acceleration Interface | N  | Y |
 
 <a name="3.4"></a>
 ## 3.4 NFVI Hardware Profile
-[RM1: 5.4 NFVI HW profiles features and requirements](../../../ref_model/chapters/chapter05.md#5.4) defines the NFVI hardware layer profiles.The labs are typically provisioned with the minimal required hardware and thus it is difficult to partition the available hardware to provision/configure multiple NFVI profiles. However, when reference implementations and the follow up testing and verification are conducted, the hardware profile need to be clearly described. This is especially important for performance testing and verification. 
+[RM1: 5.4 NFVI HW profiles features and requirements](../../../ref_model/chapters/chapter05.md#5.4) defines the NFVI hardware layer profiles.The labs are typically provisioned with the minimal required hardware and thus it is difficult to partition the available hardware to provision/configure multiple NFVI profiles. However, when reference implementations and the follow up testing and verification are conducted, the hardware profile need to be clearly described. This is especially important for performance testing and verification.
```

Comment:
The Section heading has been changed and also we have started using "Cloud infrastructure" instead of NFVI -- sorry, should have pointed out earlier.

```suggestion
[RM1: 5.4 Cloud Infrastructure Hardware Profiles features and requirements](../../../ref_model/chapters/chapter05.md#5.4) defines the Cloud Infrastructure hardware layer profiles.The labs are typically provisioned with the minimal required hardware and thus it is difficult to partition the available hardware to provision/configure multiple NFVI profiles. However, when reference implementations and the follow up testing and verification are conducted, the hardware profile need to be clearly described. This is especially important for performance testing and verification.
```